src/components/RegisterModal.jsx

```diff
@@ -1,19 +1,19 @@
 import React from "react";
 import { Button, Modal, Form } from "semantic-ui-react";
 
-function RegisterModal({onAuthenticate}) {
+function RegisterModal({ onAuthenticate }) {
   const [open, setOpen] = React.useState(false);
 
   const onCancelHandler = (event) => {
     event.preventDefault();
-    setOpen(false)
-  }
+    setOpen(false);
+  };
 
   const onAuthenticateHandler = async (event) => {
     event.preventDefault();
-    const successful = await onAuthenticate(event)
-    setOpen(!successful)
-  }
+    const successful = await onAuthenticate(event);
+    setOpen(!successful);
```

Comment:
this is not very readable, I would look over the naming standards that you have to make it easier to understand what is going on.

Reply:
We will try to be more careful with naming conventions next time. Thanks.